tck/src/main/java/org/apache/jdo/tck/pc/mylib/PCRectString.java

```diff
@@ -0,0 +1,185 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.jdo.tck.pc.mylib;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import javax.jdo.AttributeConverter;
+import java.io.Serializable;
+import java.util.Date;
+
+/**
+ * PersistenceCapable class to test JDO AttributeConverter interface.
+ * It#s fields of type Point are converted to strins in the datastore.
+ *
+ * @author Michael Bouschen
+ */
+public class PCRectString {
+    private static long counter = new Date().getTime();
+
+    private static long newId() {
+        synchronized (PCRectString.class) {
+            return counter++;
+        }
+    }
+
+    public long id = newId();
+
+    private Point upperLeft;
+    private Point lowerRight;
+
+    public PCRectString() {
+    }
+
+    public PCRectString(long id, Point ul, Point lr) {
+        upperLeft = ul;
+        lowerRight = lr;
+    }
+
+    public PCRectString(Point ul, Point lr) {
+        upperLeft = ul;
+        lowerRight = lr;
+    }
+
+    public Point getUpperLeft() {
+        return upperLeft;
+    }
+
+    public Point getLowerRight() {
+        return lowerRight;
+    }
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
+        this.id = id;
+    }
+
+    public String toString() {
+        String rc = null;
+        Object obj = this;
+        try {
+            //rc = Util.getClassName(this)
```

Comment:
What purpose does this comment serve?

Reply:
Left over when I copied PCRect to PCRectString. I will remove it.